functional-tests/test-bed/src/test/java/org/arquillian/smart/testing/ftest/simple/DisabledSmartTestingTest.java

```diff
@@ -0,0 +1,85 @@
+package org.arquillian.smart.testing.ftest.simple;
+
+import java.util.List;
+import org.arquillian.smart.testing.ftest.TestBedTemplate;
+import org.arquillian.smart.testing.ftest.testbed.testresults.TestResult;
+import org.junit.Test;
+
+import static org.arquillian.smart.testing.ftest.testbed.configuration.Criteria.AFFECTED;
+import static org.arquillian.smart.testing.ftest.testbed.configuration.Mode.SELECTING;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class DisabledSmartTestingTest extends TestBedTemplate {
+
+    @Test
+    public void should_execute_all_tests_when_smart_testing_is_disabled_irrespective_of_strategy() throws Exception {
+        // given
+        project.configureSmartTesting()
+                .executionOrder(AFFECTED)
+                .inMode(SELECTING)
+            .enable();
+
+        project
+            .applyAsCommits("Single method body modification - sysout",
+                "Inlined variable in a method");
+
+        // when
+        final List<TestResult> actualTestResults = project
+            .buildOptions()
+                .withSubProjects("config/impl-base")
```

Comment:
Interesting that I am not able to reproduce this issue locally, but that might have something to do with race conditions. Let's fix #62 then.

I understood original bug report saying

> Tests should execute as per the strategy used for ordering.

as it implies "ORDERING" mode. Thanks for explanation.